images/vm-image-builder/start_libvirtd.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+# Copyright 2023 The KubeVirt Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+chown root:kvm /dev/kvm
```

Comment:
Otherwise, we would get permission issue when use `virt-install` with `--virt-type kvm`. Other way to fix the premission issue is amending `/etc/libvirt/qemu.conf` and setting `user = "root"` and `group = "root"`

Reply:
After we set `--privileged` to run the container or set `securityContext: privileged: true` in k8s pod configuration, the user and group of /dev/kvm would be root:root by default. if we do not change the user:group of `/dev/kvm` or modify `/etc/libvirt/qemu.conf`, it would meet following error when running `virt-install`:
```
qemu-kvm: ... could not open disk image ... Permission denied
```
Do you have any suggestion on how to solve this?

Reply:
I am fine with this.